hadoop-tools/hadoop-aws/src/site/markdown/tools/hadoop-aws/index.md

```diff
@@ -233,7 +234,10 @@ needs the credentials needed to interact with buckets.
 
 The client supports multiple authentication mechanisms and can be configured as to
 which mechanisms to use, and their order of use. Custom implementations
-of `com.amazonaws.auth.AWSCredentialsProvider` may also be used.
+of `com.amazonaws.auth.AWSCredentialsProvider` may also be used. 
+However, with the upcoming upgrade to AWS Java SDK V2, these classes will need to be 
```

Comment:
Does this look ok? Just to confirm:

- We are deprecating all classes that implement `com.amazonaws.auth.AWSCredentialsProvider`. 
- We will update internal hadoop credential provider classes to implement V2's credential provider. Class names will stay the same, so no changes need to be made to configs.
- We will also update delegation token binding classes and credential providers to V2, this is a breaking change but this is ok. 
- We're using a shim for credential providers, so custom cred providers which implement V1's Credential provider will still work, but we want to encourage people to update these.